### PR TITLE
Edit profile includes updating profilePic

### PR DESCRIPTION
**Pull request by kimberlym4488**
- [x] Adds edit profile and edit profile photo routes to resolvers and typedefs.

- [x] Form includes both mutations and image preview. 

- [x] Confirmed functionality on front end and mongo db/graphQL

### CODE REVIEW

File: client/src/components/EditProfile/EditProfile.js [hunk 1]
```diff
@@ -1,82 +1,81 @@
...
+} from '@mui/material';
 
 // graphQL:
-// import { useMutation } from '@apollo/client';
-// import { ADD_ITEM } from '../../utils/mutations';
+import { useMutation } from '@apollo/client';
+import { EDIT_PROFILE, EDIT_PROFILEPHOTO } from '../../utils/mutations';
 
+// image upload
+import uploadImage from '../../utils/uploadImage';
```
Comment (jmichaelbrown8) [inline client/src/components/EditProfile/EditProfile.js:17]: Nice! Using the uploadImage util

**Review by jmichaelbrown8 — APPROVED** [review]
I just merged, so you might have to untangle mutations, resolvers, and typeDefs.

The only suggestion I would make is to use the loading and error object from the useMutation to do error handling or show a loader, otherwise looks good!

**Comment by kimberlym4488** [comment]
Thanks for the quick review J! Will make the updates and push up

**Comment by kimberlym4488** [comment]
All set! Thanks :)

**Review by kelseybrianne — APPROVED** [review]

**Pull request merged**